FABulous/FABulous.py

```diff
@@ -125,23 +125,33 @@ def main():
 
     setup_global_env_vars(args)
 
-    args.top = os.getenv("FAB_PROJ_DIR").split("/")[-1]
+    if p := os.getenv("FAB_PROJ_DIR"):
```

Comment:
You can also provide a default value for os.getenv("FAB_PROJ_DIR", args.projectDir)

But FAB_PROJ_DIR should be always set by the .env bootstrap functions:
https://github.com/FPGA-Research/FABulous/blob/FABulous2.0-development/FABulous/FABulous_CLI/helper.py#L34

Reply:
I notice that, but this makes the type checker happy.

Reply:
Okay, then let's keep it that way.